.slashrc

```diff
@@ -341,6 +344,27 @@ class MediaPlugin(slash.plugins.PluginInterface):
       slash.context.session.results.global_result.get_log_dir(), "results.xml")
     tree.write(filename)
 
+  def _load_blacklist(self, file_path = None):
+    if file_path is None or len(file_path) == 0 or not os.path.exists(file_path):
+      return
+
+    import json, glob
+    for file in glob.glob(os.path.join(str(file_path), "*.json")):
+      with open(file, 'rb') as f:
+        self.blacklist.update(json.load(f))
+
+  def _assert_not_blacklisted(self, test):
+    # Current platform and driver combine not in blacklist
+    from slash.utils.pattern_matching import Matcher
+    search= [
+      str(test.__slash__),
+      "driver=" + self._get_driver_name(),
+      "platform=" + self._get_platform_name(),
+    ]
+
+    for filter,message in self.blacklist.items():
+      assert not Matcher(filter).matches(" ".join(search)), message
```

Comment:
Can we prefix the `message` here with "Blacklisted: " so it's more clear where the assert failures come from.  Everything else looks good ;)